roles/v2v.ci/tasks/nmon_installation.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Add {{ v2v_item }}
+  add_host:
+    name: "{{ v2v_item }}"
+    ansible_user: root
+    ansible_ssh_pass: "{{ hostvars[v2v_item].root_password }}"
+    ansible_pipelining: true
+  when: (v2v_item in groups['ovirt_conversion_hosts']) or (v2v_item in groups['ovirt_conversion_host_vms'])
+
+# Install Nmon
+- name: Gathering facts
+  delegate_to: "{{ v2v_item }}"
+  setup:
+
+- set_fact:
+    nmon_file: "{% if ansible_distribution_major_version == '7' %}{{ nmon_file_rhel7 }}{% elif ansible_distribution_major_version == '8' %}{{ nmon_file_rhel8}}{% endif %}"
+
```

Comment:
I think you can do it like:
"{{ nmon_file_rhel7 if ansible_distribution_major_version == '7' elif ansible_distribution_major_version == '8' nmon_file_rhel8 }}"

Reply:
Thanks Kobi.
I preferred this way in case we will work with RHEL 9 machines, I am not sure if nmon_file_rhel8 will be good for those machines
Mordechai instructed us to align the nmon binary file by the OS it was compiled for...
Maybe I can add debug print "update the nmon file" in case of ansible_distribution_major_version == '9'?
@istein1, wdyt?

Reply:
I don't think there is a need for reminders in the code. Also I am not sure there will be a different nmon version. that will fit rhel9.
I suggest to add it to our tasks list, and if needed, we'll update it, once rhel9 will be relevant.

Reply:
np.
btw you can add rhel-9 in this way like:
"{{ nmon_file_rhel7 if ansible_distribution_major_version == '7' else nmon_file_rhel8 if ansible_distribution_major_version == '8' else  nmon_file_rhel9 if ansible_distribution_major_version == '9'}}"
to me, it's just looking cleaner if-else condition

Reply:
Thank you Ilanit & Kobi.
I also think that if-else condition is clearer than if-elif-else.
I added task to our automation tasks doc regarding "update Nmon version if needed for RHEL9"
Meanwhile I will change it to if-else